src/domUpdates.js

```diff
@@ -1,11 +1,13 @@
+import scripts from './domUpdates';
+
 let domUpdates = {
-  populateCards(recipes) {
-    let cardArea = document.querySelector('.all-cards');
+  populateCards(recipes, cardArea, userFavorites) {
     cardArea.innerHTML = '';
-    if (cardArea.classList.contains('all')) {
-      cardArea.classList.remove('all')
-    }
     recipes.forEach(recipe => {
+      let buttonStatus;
+      if (userFavorites && userFavorites.includes(recipe.id)) {
+        buttonStatus = `favorite-active`
```

Comment:
checking for favorites status is now handled here instead of the redundant method that did essentially the same thing